admin/client/src/components/FormBuilder/FormBuilder.js

```diff
@@ -250,31 +200,31 @@ export class FormBuilderComponent extends SilverStripeComponent {
    * @returns {Object}
    */
   getFieldValues() {
-    const schema = this.props.schemas[this.props.schemaUrl];
     // using state is more efficient and has the same fields, fallback to nested schema
-    const fields = (schema.state)
-      ? schema.state.fields
-      : schema.schema.fields;
-
-    // Set action
-    const action = this.getSubmitAction();
-    const values = {};
-    if (action) {
-      values[action] = 1;
+    const schema = this.props.schema.schema;
+    const state = this.props.schema.state;
+
+    if (!state) {
+      return {};
     }
 
-    // Reduce all other fields
-    return this.props.form[this.getFormId()].fields
+    return state.fields
       .reduce((prev, curr) => {
-        const match = this.findField(fields, curr.id);
+        const match = this.findField(schema.fields, curr.id);
+
         if (!match) {
           return prev;
         }
 
+        // Skip non-data fields
+        if (match.type === 'Structural') {
```

Comment:
I've changed this to `if (match.type === 'Structural' && typeof curr.value === 'undefined')` and started another discussion thread about it on this PR